todo-app/src/TodoList.js

```diff
@@ -2,32 +2,36 @@ import React, { useState } from "react";
 import "./style.css";
 
 function TodoList() {
-  const [todos, setTodos] = useState([]);
-  const [progress, setProgress] = useState([]);
-  const [done, setDone] = useState([]);
+  const [todoList, setTodoList] = useState([]);
+  const [inProgressList, setInProgressList] = useState([]);
+  const [doneList, setDoneList] = useState([]);
 
-  const [inputValue, setInputValue] = useState("");
+  const [inputValue, setInputValue] = useState([]);
```

Comment:
why input value is initialized to an array. value is expected to be a text/string value

Reply:
changes done